antora-ui/gulp.d/tasks/build.js

```diff
@@ -244,19 +244,13 @@ function getAllTasks (opts, sourcemaps, postcssPlugins, preview, src) {
 
     In CI, the file won't be available so it always
     uses the most recent version.
-
-    CSS Branch handling:
-    - master: Uses production version of boostlook styling
-    - develop: Uses Tino Agency's redesign work for staging deployment
  */
 async function fetchBoostlookCss () {
   log('Fetching boostlook.css file...')
   const skipBoostlook = process.argv.includes('--skip-boostlook')
   const cssDir = ospath.join(__dirname, '..', '..', 'src', 'css')
   const cssFilePath = ospath.join(cssDir, 'boostlook.css')
-  const boostlookBranch = process.env.BOOSTLOOK_BRANCH || 'master'
-  const sourceFilename = boostlookBranch === 'develop' ? 'boostlook_tino.css' : 'boostlook.css'
-  const url = `https://raw.githubusercontent.com/boostorg/boostlook/${boostlookBranch}/${sourceFilename}`
+  const url = 'https://raw.githubusercontent.com/boostorg/boostlook/master/boostlook.css'
```

Comment:
Before recent changes, going back to Alan's code, this section was:

```
  const boostlookBranch = process.env.BOOSTLOOK_BRANCH || 'master'
  const url = `https://raw.githubusercontent.com/cppalliance/boostlook/${boostlookBranch}/boostlook.css`
```

How about reverting to that?  It allows flexibility.  You never know when it might be necessary to switch to `develop` again, and a variable allows that.